mesonpy/_compat.py

```diff
@@ -37,6 +37,17 @@
     cached_property = lambda x: property(functools.lru_cache(maxsize=None)(x))  # noqa: E731
 
 
+if sys.version_info >= (3, 9):
+    import importlib.resources
+
+    def read_binary(package: str, resource: str) -> bytes:
+        return importlib.resources.files(package).joinpath(resource).read_bytes()
+else:
+    import importlib.resources
```

Comment:
This is the same import twice in both `if` and `else` branches, so it can just be at the top of the file?

Reply:
Ah, of course :sweat_smile: